lib/kube/proxy/forwarder.go

```diff
@@ -957,7 +957,7 @@ func (f *Forwarder) getKubeAccessDetails(
 		// accessChecker.CheckKubeGroupsAndUsers returns the accumulated kubernetes_groups
 		// and kubernetes_users that satisfy te provided matchers.
 		// When a KubernetesResourceMatcher, it will gather the Kubernetes principals
-		// whose role satisfy the the desired Kubernetes Resource.
+		// whose role satisfy the desired Kubernetes Resource.
```

Comment:
```suggestion
		// with roles that satisfy the desired Kubernetes Resource.
```

Reply:
Thank you for your patient guidance. It has been fixed. Please take a look